ASM/src/hacks.asm

```diff
@@ -351,6 +692,28 @@ Gameplay_InitSkybox:
 ; Freestanding models
 ;==================================================================================================
 
+
+;Replaces:
+;   who knows ; Draw Rupee Function
```

Comment:
I hope whatever was there is being replaced completely so it's not important, lol